meteor/client/ui/RundownList/RundownListItemView.tsx

```diff
@@ -110,6 +111,9 @@ export default withTranslation()(function RundownListItemView(props: Translated<
 			<span className="rundown-list-item__text">
 				<JonasFormattedTime timestamp={rundown.modified} t={t} />
 			</span>
+			<span className="rundown-list-item__text">
```

Comment:
I would very much like this to be conditional on if there are any RundownShelfLayoutSelections to be displayed. Some users don't use this feature at all and will just see a wall of "Default's" for no reason.